Misc/NEWS.d/next/Library/2020-01-31-09-32-19.bpo-39505.drA2Qa.rst

```diff
@@ -0,0 +1,5 @@
+When windows users use "python -m venv ENV_DIR", a python virtual environment will be created in ENV_DIR.

```

Comment:
I'm not sure a NEWS entry is needed for a bug-fix patch like this. I'll add the "skip-news" label so that you can remove this.